### DISCUSSION TITLE
vsce package problems

### DISCUSSION THREAD

**Discussion by grafandreas** [eclipse-langium/langium#1751] [Q&A]
I once was successfully using vsce package, now having two new projects, I get the following problem (tried with default hello-world):

 ERROR  Command failed: npm list --production --parseable --depth=99999 --loglevel=error
npm error code ELSPROBLEMS
npm error missing: hello-word@file:, required by hello-word@0.0.1

Has anyone encountered this?

**Comment by msujew** [comment]
Hey @grafandreas,

vsce isn't playing nice with npm workspaces. See also https://github.com/microsoft/vscode-vsce/issues/580. Essentially, it's a false-positive in a unnecessary verification. You can skip this verification by adding `--no-dependencies` to the `vsce` call. Alternatively, you can also configure this into your package.json. See https://github.com/microsoft/vscode-vsce/issues/580#issuecomment-1399955227.
